spec/models/enterprise_spec.rb

```diff
@@ -145,6 +145,86 @@
       it "sets the enterprise contact to the owner by default" do
         expect(enterprise.contact).to eq enterprise.owner
       end
+
+      context "prevent an wrong instagram link pattern" do
+        it "expects to be invalid the instagram attribute https://facebook.com/user" do
+          e = build(:enterprise, instagram: 'https://facebook.com/user')
+          expect(e).to_not be_valid
+        end
+
+        it "expects to be invalid the instagram attribute tagram.com/user" do
+          e = build(:enterprise, instagram: 'tagram.com/user')
+          expect(e).to_not be_valid
+        end
+
+        it "expects to be invalid the instagram attribute https://instagram.com/user/preferences" do
+          e = build(:enterprise, instagram: 'https://instagram.com/user/preferences')
+          expect(e).to_not be_valid
+        end
+      end
+
+      context "accepted pattern" do
+        it "expects to be valid empty instagram attribute" do
+          e = build(:enterprise)
+          expect(e).to be_valid
+        end
+
+        it "expects to be valid the instagram attribute @my-user" do
+          e = build(:enterprise, instagram: '@my-user')
+          expect(e).to be_valid
```

Comment:
It would be nice to test the result here as well.
```suggestion
          expect(e).to be_valid
          expect(e.instagram).to eq "my-user"
```

All valid cases should be tested this way. Otherwise this method would pass the test as well:
```rb
  def instagram=(value)
    self[:instagram] = ""
  end
```

Reply:
Sorry for the slight absence. Currently working on a fix to the spec file and adding the suggested changes!

This is a very good point, and that should also help me figure out the issue with that enterprise test. Thanks!